spec/index.html

```diff
@@ -1012,16 +1012,12 @@ <h2>RDF Datasets</h2>
       can avoid interoperability issues by not ascribing importance to
       the presence or absence of empty named graphs.</p>
 
-    <p>SPARQL 1.2 [[SPARQL12-CONCEPTS]] also defines the concept of an RDF
-			Dataset.  The definition of an RDF Dataset in SPARQL 1.1 and this
-			specification differ slightly in that this specification allows RDF
-			Graphs to be identified using either an IRI or a blank node.  SPARQL 1.1
-			Query Language only allows RDF Graphs to be identified using an IRI.
-			Existing SPARQL implementations might not allow blank nodes to be used
-			to identify RDF Graphs for some time, so their use can cause
-			interoperability problems.
-      <a href="#section-skolemization">Skolemizing</a> blank nodes used as
-      graph names can be used to overcome these interoperability problems.</p>
+    <p>SPARQL 1.2 [[SPARQL12-QUERY]] uses 
+      <a data-cite="SPARQL12-QUERY#sparqlDataset">the same concept of an RDF Dataset</a>
+      in which the graph names of named graphs may be IRIs or blank nodes.
```

Comment:
`SPARQL 1.2 [[SPARQL12-QUERY]] uses `
`<a data-cite="SPARQL12-QUERY#sparqlDataset">the same concept of an RDF Dataset</a>` 
as ... what/where/when? ...,
`in which the graph names of named graphs may be IRIs or blank nodes.`

Reply:
"RDF Dataset" is defined immediately above the note so I think it is clear.